### PR TITLE
Upgrading to Bevy 0.12

### PR DESCRIPTION
**Pull request by rszemplinski**
Not saying the changes I made are the best way to make this work with Bevy `0.12` but they do compile and run. :sweat_smile:

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 /target
 Cargo.lock
-.vscode
\ No newline at end of file
+.vscode
```
Comment (Shatur) [inline .gitignore:3]: I would suggest to remove additional changes from this file.

File: examples/depth_bias.rs [hunk 2]
```diff
@@ -23,6 +24,8 @@ fn main() {
         .insert_resource(ClearColor(Color::BLACK))
         .insert_resource(Msaa::Sample4)
         .add_plugins(DefaultPlugins)
+        .add_plugins(FrameTimeDiagnosticsPlugin::default())
+        .add_plugins(LogDiagnosticsPlugin::default())
```
Comment (Shatur) [inline examples/depth_bias.rs:28]: Why you includes these plugins into examples?

File: src/lib.rs [hunk 3]
```diff
@@ -14,20 +14,14 @@ pub mod prelude {
     pub use crate::PolylinePlugin;
 }
 
-pub const SHADER_HANDLE: HandleUntyped =
-    HandleUntyped::weak_from_u64(Shader::TYPE_UUID, 12823766040132746065);
+pub const SHADER_HANDLE: Handle<Shader> =
```
Comment (Shatur) [inline src/lib.rs:17]: Could you use `embed_asset!`? See https://bevyengine.org/news/bevy-0-12/#embedded-assets

File: src/material.rs [hunk 4]
```diff
@@ -152,27 +151,28 @@ impl RenderAsset for PolylineMaterial {
...
-            AlphaMode::Opaque
-        };
-
-        Ok(GpuPolylineMaterial {
-            buffer,
-            perspective: material.perspective,
-            alpha_mode,
-            bind_group,
-        })
+            panic!();
```
Comment (Shatur) [inline src/material.rs:174]: Why do you panic here?

**Review by Shatur — CHANGES_REQUESTED** [review]

File: Cargo.toml [hunk 5]
```diff
@@ -17,22 +17,22 @@ authors = [
...
+bevy = { version = "0.12.0", default-features = false, features = [
     "bevy_core_pipeline",
     "bevy_render",
     "bevy_asset",
 ] }
 
 [dependencies.naga]
 features = ["glsl-in", "spv-out", "wgsl-out"]
-version = "0.12"
+version = "0.14"
```
Comment (Shatur) [inline Cargo.toml:29]: Could you set it to 0.13? This is what bevy use.

**Review by Shatur — CHANGES_REQUESTED** [review]

**Comment by IceSentry** [comment]
closed by #53

**Pull request closed without merging**